Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -34,6 +34,7 @@
 #include "SplineConverter.h"
 
 #include "BlueFramework/Core/Diagnostics/log.h"
+#include <fstream>

```

Comment:
Why do we need fstream here?

Reply:
I forgot to delete this line after I tested some functionality.  
Resolved: [aac48ed](https://github.com/tumcms/Open-Infra-Platform/pull/539/commits/aac48ed9f897985601b33b741f9ace040cb8f854)